src/xarray_regrid/methods/conservative.py

```diff
@@ -124,7 +131,7 @@ def conservative_regrid_dataarray(
 
             data = data.transpose(coord, ...)
             data = apply_weights(data, weights, coord, target_coords)
-
+    data.attrs = attrs
```

Comment:
Add coordinates' attributes:
```suggestion
    for dcrd, _attr in zip(datacoords, dcrds_attrs, strict=True):
        dcrd.attrs = _attr
    data.attrs = attrs
```